modules/core/src/main/java/org/apache/ignite/internal/visor/verify/VisorValidateIndexesTaskArg.java

```diff
@@ -55,11 +58,13 @@ public VisorValidateIndexesTaskArg() {
     /**
      * @param caches Caches.
      */
-    public VisorValidateIndexesTaskArg(Set<String> caches, Set<UUID> nodes, int checkFirst, int checkThrough) {
+    public VisorValidateIndexesTaskArg(
```

Comment:
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines check Method Arguments

Reply:
The indents fixed in the next commit